pkg/controller/quayecosystem/configuration/reconcile.go

```diff
@@ -442,13 +545,18 @@ func (r *ReconcileQuayEcosystemConfiguration) quayRegistryStorage(meta metav1.Ob
 func (r *ReconcileQuayEcosystemConfiguration) quayDeployment(meta metav1.ObjectMeta) error {
 
 	meta.Name = resources.GetQuayResourcesName(r.quayEcosystem)
+	resources.BuildQuayResourceLabels(meta.Labels)
 
 	quayDeploymentPodSpec := corev1.PodSpec{
 		Containers: []corev1.Container{{
 			Image: r.quayEcosystem.Spec.Quay.Image,
 			Name:  meta.Name,
 			Ports: []corev1.ContainerPort{{
-				ContainerPort: 80,
+				ContainerPort: 8080,
+				Name:          "http",
+			}, {
+				ContainerPort: 8080,
+				Name:          "https",
```

Comment:
Was 8080 for https instead of 8443 purposeful?

Reply:
@thomasmckay copy/paste error. Updated